src/pages/workspace/expensifyCard/issueNew/ConfirmationStep.tsx

```diff
@@ -33,9 +37,12 @@ function ConfirmationStep({policyID, backTo}: ConfirmationStepProps) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
     const {isOffline} = useNetwork();
-
+    const [account] = useOnyx(ONYXKEYS.ACCOUNT);
     const [issueNewCard] = useOnyx(ONYXKEYS.ISSUE_NEW_EXPENSIFY_CARD);
-
+    const [pendingContactAction] = useOnyx(ONYXKEYS.PENDING_CONTACT_ACTION);
+    const contactMethod = UserUtils.getContactMethod();
+    const validateLoginError = issueNewCard?.errors;
```

Comment:
AFAIR, Sometimes server return errors as an array, so let's use this instead:
```suggestion
    const validateLoginError = ErrorUtils.getLatestErrorMessageField(issueNewCard);
```